tests/phpunit/api/v3/MultilingualTest.php

```diff
@@ -59,12 +59,12 @@ public function testOptionLanguage($version) {
 
     CRM_Core_I18n_Schema::addLocale('fr_CA', 'en_US');
 
-    $this->callAPISuccess('Setting', 'create', array(
-      'languageLimit' => array(
-        'en_US',
-        'fr_CA',
-      ),
-    ));
+    $this->callAPISuccess('Setting', 'create', [
+      'languageLimit' => [
+        'en_US' => 1,
+        'fr_CA' => 1,
+      ],
```

Comment:
Test was setting `languageLimit` in the wrong format (but passing anyway due to a synergistic bug in php).

Arguably the format used by `languageLimit` is ridiculous and likely to lead to more mistakes like this, but for now we're stuck with it.